src/extension/_locales/de/messages.json

```
@@ -117,7 +117,7 @@
     "message": "Ungültige Teilen-URL"
   },
   "config_invalid_giturl": {
-    "message": "Ungültige Repository-URL"
+    "message": "Ungültige Repository-URL, muss beginnen mit https://github.com"
```

I think I would change this to 
```suggestion
    "message": "Ungültige Repository-URL, muss mit https://github.com beginnen"
```